app/core/src/main/java/com/fsck/k9/AccountPreferenceSerializer.kt

```diff
@@ -441,6 +439,10 @@ class AccountPreferenceSerializer(
         editor.remove("$accountUuid.markMessageAsReadOnView")
         editor.remove("$accountUuid.markMessageAsReadOnDelete")
         editor.remove("$accountUuid.alwaysShowCcBcc")
+
+        // TODO: to leave this in or not? On one hand, if I don't leave it, this value will be "orphaned" and just hang
+        // around on the device once we remove this setting. OTOH, it may be overkill to keep this reference to an
+        // unused boolean value.
```

Comment:
Remove the line of code. That's what we have done in the past.

I do have plans to change the way we store settings. Once implemented it will also allow us to get rid of such orphaned settings.